src/components/StaffComments/StaffComments.vue

```diff
@@ -0,0 +1,256 @@
+            Save
+          </v-btn>
+          <v-btn text color="primary" @click="close()">
+            Cancel
+          </v-btn>
+        </div>
+      </v-card-actions>
+
+      <v-card-text class="mt-6 pa-0">
+        <div class="existing-comments pr-5">
+          <div v-for="(comment, i) in comments" :key="i" class="body-2">
+            <p class="pre-line" v-html="comment.comment" />
+            <p class="font-italic">
+              {{ comment.submitterDisplayName }}
+              &hyphen;
+              {{apiToSimpleDateTime(comment.timestamp)}}
+            </p>
+          </div>
+        </div>
+      </v-card-text>
+    </v-card>
+  </v-menu>
+</template>
+
+<script lang="ts">
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import { CommentIF, FormIF } from '@/interfaces'
+import { DateMixin, FilingMixin } from '@/mixins'
+
+@Component({})
+export default class StaffComments extends Mixins(DateMixin, FilingMixin) {
+  $refs!: Vue['$refs'] & {
+    textarea: FormIF
+  }
+
+  /** Axios instance prop. */
+  @Prop({ required: true })
+  readonly axios: any
+
+  /** Business ID prop. */
+  @Prop({ required: true })
+  readonly businessId: string
+
+  /** Max Length prop (optional). */
+  @Prop({ default: 4096 })
+  readonly maxLength: number
+
+  /** Model property for v-menu (ie, whether to show the dialog). */
+  private showComments = false
+
+  /** The list of comments. */
+  private comments: Array<CommentIF> = []
+
+  /** The new comment's text. */
+  private comment: string = null
+
+  /** Whether a comment is being saved. */
+  private isSaving = false
+
+  /** The number of chars remaining in the new comment. */
+  private get charsRemaining (): number {
+    const length = this.comment ? this.comment.length : 0 // comment may be null
+    return (this.maxLength - length)
+  }
+
+  /** The Number of Comments string for this entity. */
+  private get numComments (): string {
+    const numComments = this.comments.length
+    return (numComments === 1 ? '1 Comment' : `${numComments} Comments`)
+  }
+
+  /** Array of validations rules for the textarea. */
+  private get rules (): Array<Function> {
+    // exclude whitespace in minimum length check
+    // include whitespace in maximum length check
+    return [
+      val => (val && val.trim().length > 0) || 'Enter a comment.',
+      val => (val && val.length <= this.maxLength) || 'Maximum characters reached.'
+    ]
+  }
+
+  /** Called when the component is created. */
+  private async created (): Promise<void> {
+    await this.fetchStaffComments()
+  }
+
+  /** Fetches the staff comments from the API. */
+  private async fetchStaffComments (): Promise<void> {
+    const url = `businesses/${this.businessId}/comments`
+    this.comments = await this.axios.get(url)
+      .then(res => this.flattenAndSortComments(res && res.data && res.data.comments))
+      .catch(() => [])
+  }
```

Comment:
Were you able to make any mocked requests in Storybook?
No action required here, just curious because i'd like to know how :)

Reply:
Yes!

Reply:
Very cool, i wonder if my troubles were due too using the `SbcFeeSummary` common component imported into the shared component, where the actual requests were from that import.. 🤔 

Either way, well done!